src/Engine/EmitMSIL/ExecuteIL.cs

```diff
@@ -60,7 +60,7 @@ class ExecuteIL
             key = CodeGenIL.KeyGen(className, methodName, args.Count);
             mi = methodCache[key];
             // 6. Emit call to ReplicationLogic by passing input args from previous steps
-            var d = Replication.ReplicationLogic(mi, args, guides);
+            var d = Replication.ReplicationLogic(null, args, guides, null);
```

Comment:
This code was initially written as an example so I could wrap around my head over what was being emitted by replicating it as actual C# code. Now it looks like this example isn't relevant anymore unless we want to instantiate CLRFunctionEndPoints and all those types. So I suggest removing it if it doesn't hold value anymore.

Reply:
I can remove the entire File and the Program.cs file too